mrp_subcontracting_skip_no_negative/tests/test_mrp_subcontracting_skip_no_negative.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo.exceptions import ValidationError
+from odoo.tests import Form, common
+
+
+class TestMrpSubcontractingSkipNoNegative(common.TransactionCase):
```

Comment:
Inherit TestMrpSubcontractingCommon from mrp_subcontracting to save lines on the setup?